### PR TITLE
Remove an auto-link

### PR DESCRIPTION
**Pull request by teoli2003**
Just an auto-link (a page that links to itself)

### CODE REVIEW

File: files/en-us/web/html/element/a/index.md [hunk 1]
```diff
@@ -449,9 +449,7 @@ Spacing may be created using CSS properties like {{CSSxRef("margin")}}.
           href="/en-US/docs/Web/HTML/Content_categories#interactive_content"
           >interactive content</a
         > or an
-        <a href="/en-US/docs/Web/HTML/Element/a"
-          >a</a
-        > element, and no descendant may have a specified
+        <code>>a</code> element, and no descendant may have a specified
```
Comment (bsmth) [inline files/en-us/web/html/element/a/index.md:452]: small leftover?

```suggestion
        <code>a</code> element, and no descendant may have a specified
```

File: files/en-us/web/html/element/a/index.md [hunk 2]
```diff
@@ -449,9 +449,7 @@ Spacing may be created using CSS properties like {{CSSxRef("margin")}}.
           href="/en-US/docs/Web/HTML/Content_categories#interactive_content"
           >interactive content</a
         > or an
-        <a href="/en-US/docs/Web/HTML/Element/a"
-          >a</a
-        > element, and no descendant may have a specified
+        <code>a</code> element, and no descendant may have a specified
```
Comment (bsmth) [inline files/en-us/web/html/element/a/index.md:452]: Sorry, my mistake, we now have "an `a` element" and I presume we need "an <code>&lt;a&gt;</code> element":

```suggestion
        <code>&lt;a&gt;</code> element, and no descendant may have a specified
```

There's another one on L469 if you want to get it in this PR.

Comment (teoli2003) [reply files/en-us/web/html/element/a/index.md:452]: Oh yes you are right, let's fix them!

**Review by bsmth — APPROVED** [review]
Looks good, thank you!

**Pull request merged**